solutions/cf-863B.mdx

```diff
@@ -48,5 +48,45 @@ int main() {
 ```
 
 </CPPSection>
+	
+<JavaSection>
+	
+```java
+import java.util.*;
+import java.io.*;
+
+public class Kayaking {
+	public static void main (String[] args) {
+		Kattio io = new Kattio();
+		int N = io.nextInt() * 2;
+		int[] W = new int[N];
+		for (int x = 0; x < W.length; x++) {
+			W[x] = io.nextInt();
+		}
+		Arrays.sort(W);
+		int minInstability = Integer.MAX_VALUE;
+		for (int i = 0; i < N; i++) {
+			for (int j = i + 1; j < N; j++) {
+				ArrayList<Integer> list = new ArrayList<>();
```

Comment:
idk much about java but [Change to `List<>`](https://usaco.guide/general/adding-solution?lang=java#java)?

@HarryWangATX

Reply:
[yep](https://stackoverflow.com/questions/2279030/type-list-vs-type-arraylist-in-java)

Reply:
ye

Reply:
https://usaco.guide/general/adding-solution/#general